test/test_model.cfg

```diff
@@ -121,4 +121,128 @@
       "hamiltonian_func" : "x_drive"
     }
   }
+  Tasks:
+  {
+    init_ground:
+    {
+      c3type: InitialiseGround
+      params:
+      {
+        init_temp:
+        {
+          value: 0.05
+          min_val: -0.001
+          max_val: 0.22
+          unit: K
+          symbol: \alpha
+        }
+      }
+    }
```

Comment:
Including tasks here causes the QASM test to fail, since that now tries to create a thermal state. Up to now, we only implemented a density vector solver for the Lindblad master equation, so we get an error about a not implemented Von Neumann eq. The fix is easy: Just make a separate `test_model_spam.cfg` and use that in the tests for the Tasks.